mono/mini/mini-runtime.c

```diff
@@ -4749,13 +4749,17 @@ print_jit_stats (void)
 void
 mini_cleanup (MonoDomain *domain)
 {
+	if (mono_jit_stats.enabled)
```

Comment:
I think ideally we check `mono_stats.enabled`, even if currently that's only set at the same point as `mono_jit_stats`.